{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/pagination/pagination.html

```diff
@@ -0,0 +1,38 @@
+{% if page_obj.has_other_pages() %}
+  <nav class="pgn-Pagination" aria-label="You are on page {{ page_obj.number }}">
+    {% if page_obj.has_previous() %}
+      <a href="{{ get_pagination_url(page_obj.previous_page_number()) }}" class="pgn-Pagination_Control pgn-Pagination_Control-prev" aria-label="Previous page">
+        {% include 'chevron-right.svg' %}
+      </a>
+    {% endif %}
+
+    {# using a separate element and aria-hidden because screen readers will
+    probably only care about previous/next #}
+    <div class="pgn-Pagination_Pages" aria-hidden="true">
+
+      {% for page_num in page_numbers_adjusted %}
+        {% if not page_num %}
+          <span class="pgn-Pagination_Page pgn-Pagination_Page-fake">
+            ...
+          </span>
+        {% elif page_num == page_obj.number %}
+          <span class="pgn-Pagination_Page pgn-Pagination_Page-current">
+            {{ page_num }}
+          </span>
+        {% else %}
+          <a class="pgn-Pagination_Page {% if page_num == page_obj.number %}pgn-Item_Action-current{% endif %}"
+             href="{{ get_pagination_url(page_num) }}"
+             aria-current="{% if page_num == page_obj.number %}true{% else %}false{% endif %}">
```

Comment:
Same thing as above - this condition cannot be true. (And it makes little sense to have ARIA attributes on a block with `aria-hidden` - probably a hangover from the fact we were previously targeting this attribute rather than a class name as we do now.)